libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/route/NavigationMapRoute.java

```diff
@@ -445,7 +445,7 @@ private void addDirectionWaypoints() {
   }
 
   private void addUpcomingManeuverArrow(RouteProgress routeProgress) {
-    if (routeProgress.upcomingStepPoints() == null || routeProgress.upcomingStepPoints().size() < TWO_MANEUVERS) {
+    if (routeProgress.upcomingStepPoints() == null || routeProgress.upcomingStepPoints().size() < TWO_MANEUVERS || routeProgress.currentStepPoints().size() < TWO_MANEUVERS) {
```

Comment:
For readability, can we create a new `int` constant here called `TWO_POINTS` that is used for:
```
routeProgress.currentStepPoints().size() < TWO_POINTS
```

Reply:
Sure, not a problem, this seems to make the TWO_MANUVERS variable unused though, so just renamed it, I hope that's fine.